dashboard/app/models/user.rb

```diff
@@ -721,6 +721,14 @@ def secret_word_account?
     sections_as_student.any? {|section| section.login_type == Section::LOGIN_TYPE_WORD}
   end
 
+  # True if the account is teacher-managed, is in at least one picture section, and
+  # is not in any non-picture sections
+  def secret_picture_account_only?
+    return false unless teacher_managed_account?
```

Comment:
might be even easier to follow if we just make the first line
```
return false unless teacher_managed_account? && sections_as_student.present?
```
another simple way to express this could be
```
sections_as_student.map(&:login_type).uniq == [Section::LOGIN_TYPE_PICTURE]
```


Really any of these is probably fine.

Reply:
`.present?` doesn't seem to work on arrays (we want to check that the array has elements, and we have a style rule preventing checking the length).

The second option is interesting, though, I'll see if that works right.